alchemist/alchemist-grid/build.gradle

```diff
@@ -0,0 +1,10 @@
+dependencies {
+    compile project(':alchemist-implementationbase')
+    compile project(':alchemist-interfaces')
+    compile project(':alchemist-loading')
+    compile project(':alchemist-engine')
+    testCompile project(':alchemist-incarnation-sapere')
+	compile group: 'org.apache.ignite', name: 'ignite-core', version: '2.2.0'
+    compile group: 'org.apache.ignite', name: 'ignite-spring', version: '2.2.0'
```

Comment:
Do not hardcode versions. Use the `gradle.property` file in the parent project to store dependencies versions